### DISCUSSION TITLE
Fast Stable Diffusion/Gradio.live - 504 Gateway Time-out

### DISCUSSION THREAD

**Discussion by meunumerotim333** [TheLastBen/fast-stable-diffusion#1495] [Q&A]
fast stable diffusion isn't connecting on gradio.live, tries to load and gives 504 Gateway Time-out.

It's a general problem? 

I'm using 1.5 model 

It showed something that normally doesn't show:
_Checkpoint model.ckpt [4c86efd062] not found; loading fallback model.ckpt [4c86efd060]_

Using this copy: https://colab.research.google.com/github/TheLastBen/fast-stable-diffusion/blob/main/fast_stable_diffusion_AUTOMATIC1111.ipynb

I've tried delete the sd folder and the copy on notebook and restart all again.
I've also tried using another email. Both failed

**Comment by meunumerotim333** [comment]
Print Screens:
https://i.imgur.com/4Lf1MOJ.png
https://i.imgur.com/VFI3dB9.png

**Comment by vesperialuna** [comment]
i have the same issue as well currently. 504 gateway error. didn't have this issue when we were on the .app domain. the .live one is a bit problematic.

**Reply by meunumerotim333** [reply]
Seems the recently update have fix it

**Reply by 70pice** [reply]
This problem have happen again

**Reply by Durogaff** [reply]
Ok, so you too. Was thinking it was something i may have done.
